doc/api/stream.md

```diff
@@ -770,10 +774,16 @@ cause some amount of data to be read into an internal buffer.
 
 ```javascript
 const readable = getReadableStreamSomehow();
-readable.on('readable', () => {
+readable.on('readable', function() {
```

Comment:
Nit: not sure why this was changed and it doesn't really matter but for consistency I would keep the arrow function.

Reply:
@lpinca it is using this.read() below

Reply:
Oops, ignore me.